.github/workflows/build.yml

```diff
@@ -208,6 +208,49 @@ jobs:
         asset_content_type: application/zip
 
 
+  build_macos_legacy:
+    runs-on: macos-latest
+    needs: create_release
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install Python
+      # We need the official Python, because the GA ones only support newer macOS versions
+      env:
+        PYTHON_VERSION: 3.10.5
```

Comment:
uh, remembering to bump this every 2 months is gonna be annoying https://peps.python.org/pep-0619/

Is there any way to make it download "the latest 3.10"?

Reply:
> If it's difficult, don't worry about it

If it's difficult then I have an idea (it's quite hacky, I know):
If Python 3.10.7 exists, abort the workflow (so that people notice it)
If python 3.10.6 exists, use that and return a warning
Otherwise, use Python 3.10.5

Reply:
That's just my (pretty shitty) idea__